examples/mobilenetv3/README.md

```diff
@@ -0,0 +1,23 @@
+## About sample image
+
+A sample image (image file in `218`) was obtained from the ImageNet dataset.
```

Comment:
I don't think we can distribute ImageNet images.  The ImageNet terms and conditions state that the data can only be shared with people who agree to the terms and conditions first, and we can't guarantee that.  This will have to be replaced with an image that we have the right to distribute under this project's license (Apache 2.0). If you can provide a photo that you take yourself of a non-copyrighted subject, I think that should work.

Reply:
The photo was replaced to that I've taken. Could you check this again?